perun-base/src/test/resources/test-schema.sql

```diff
@@ -1156,6 +1156,7 @@ create table tasks (
 	start_time timestamp,
 	end_time timestamp,
 	engine_id integer,
+	schedule_expr varchar(256),
```

Comment:
DB version needs to be updated also at the end of this file.

Reply:
Fixed.